### PR TITLE
Update gateway onboarding docs

### PR DESCRIPTION
**Pull request by mjamescompton**
#### Summary

References https://github.com/TheThingsNetwork/lorawan-stack/issues/3947

#### Screenshots

Screenshot 2022-07-11 at 14 36 11
Screenshot 2022-07-11 at 14 36 16
Screenshot 2022-07-11 at 14 36 22

Image optimisation

Screenshot 2022-07-11 at 14 32 37

#### Changes

- Extend the current docs to include claiming option.
- Remove out of date video and related text.
- Optimise all the gateway related images.

#### Notes for Reviewers

@benolayinka Not sure what your process is for making the videos but if you have time you can create a new one via your local if you run the PR https://github.com/TheThingsIndustries/lorawan-stack/pull/3304 . I still need it approved so it may change so I'll leave it up to you weather you want to wait until this is live to update the video docs.

Also I will wait until pr is approved to merge.

#### Checklist

- [ ] Scope: The referenced issue is addressed, there are no unrelated changes.
- [ ] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [ ] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/gateways/adding-gateways/_index.md [hunk 1]
```diff
@@ -25,18 +25,27 @@ Adding gateways using the Console or the CLI is usually most convenient, so thos
...
 
-Go to **Gateways** in the top menu, and click **+ Add Gateway** to reach the gateway registration page.
 
-Fill the **Gateway ID**, **Gateway EUI** (if your gateway has an EUI) and **Frequency Plan**. The other fields are optional. Click **Create Gateway** to finish.
+
+Fill the **Gateway EUI** and click **Confirm** or **Continue without EUI**.
+
+Depending of weather the **Gateway EUI** is claimable you will either be shown the claiming form or the manual registration form.
+
+{{< note >}}Claiming may not be possible on your version of The Thing Stack, if not you will see the manual registration form by default.{{</ note >}}
```
Comment (kschiffer) [inline doc/content/gateways/adding-gateways/_index.md:37]:
```suggestion
{{< note >}}Claiming of gateways is not supported on {{% tts %}} Open Source. In that case, you will see the manual registration form by default.{{</ note >}}
```

File: doc/content/gateways/adding-gateways/_index.md [hunk 2]
```diff
@@ -25,18 +25,27 @@ Adding gateways using the Console or the CLI is usually most convenient, so thos
...
-{{< youtube "o6nwc3APXns" >}}
-</details>
 
-Go to **Gateways** in the top menu, and click **+ Add Gateway** to reach the gateway registration page.
 
-Fill the **Gateway ID**, **Gateway EUI** (if your gateway has an EUI) and **Frequency Plan**. The other fields are optional. Click **Create Gateway** to finish.
+
+Fill the **Gateway EUI** and click **Confirm** or **Continue without EUI**.
+
+Depending of weather the **Gateway EUI** is claimable you will either be shown the claiming form or the manual registration form.
```
Comment (kschiffer) [inline doc/content/gateways/adding-gateways/_index.md:35]:
```suggestion
Depending on whether the **Gateway EUI** is claimable you will either be shown the claiming form or the manual registration form.
```

File: doc/content/gateways/adding-gateways/_index.md [hunk 3]
```diff
@@ -25,18 +25,27 @@ Adding gateways using the Console or the CLI is usually most convenient, so thos
...
+
+{{< note >}}Claiming may not be possible on your version of The Thing Stack, if not you will see the manual registration form by default.{{</ note >}}
+
+{{< figure src="gateway-add.png" alt="Gateway add" >}}
+
+On the manual registration form fill in the **Frequency Plan** and the **Gateway ID** if it was not pre filled, The other fields are optional. Click **Register Gateway** to finish.
 
 {{< figure src="gateway-creation.png" alt="Gateway creation" >}}
 
+On claiming form fill in the **Authentication Code**, **Gateway ID** and the **Frequency Plan**. Click **Claim Gateway** to finish.
```
Comment (kschiffer) [inline doc/content/gateways/adding-gateways/_index.md:45]:
```suggestion
If your gateway is claimable (e.g. The Things Indoor Gateway), you will see the claiming form. Here, fill in the **Authentication Code**, **Gateway ID** and the **Frequency Plan**. Click **Claim Gateway** to finish.
```

File: doc/content/gateways/adding-gateways/_index.md [hunk 4]
```diff
@@ -25,18 +25,27 @@ Adding gateways using the Console or the CLI is usually most convenient, so thos
...
 
-<details><summary>Show video</summary>
-{{< youtube "o6nwc3APXns" >}}
-</details>
 
-Go to **Gateways** in the top menu, and click **+ Add Gateway** to reach the gateway registration page.
 
-Fill the **Gateway ID**, **Gateway EUI** (if your gateway has an EUI) and **Frequency Plan**. The other fields are optional. Click **Create Gateway** to finish.
+
+Fill the **Gateway EUI** and click **Confirm** or **Continue without EUI**.
```
Comment (kschiffer) [inline doc/content/gateways/adding-gateways/_index.md:33]:
```suggestion
Fill the **Gateway EUI** and click **Confirm**. Some gateways do not use a **Gateway EUI** (e.g. The Things Kickstarter Gateway), in which case you can just click on **Continue without EUI**.
```

**Review by kschiffer — CHANGES_REQUESTED** [review]
Thanks for this

**Review by kschiffer — APPROVED** [review]

**Review by KrishnaIyer — APPROVED** [review]
Nice!

**Pull request merged**